freqtrade/exchange/__init__.py

```diff
@@ -107,6 +109,15 @@ def id(self) -> str:
         """exchange ccxt id"""
         return self._api.id
 
+    def set_sandbox(self, api, exchange_config: dict, name: str):
+        if exchange_config.get('sandbox'):
+            if api.urls.get('test'):
+                api.urls['api'] = api.urls['test']
```

Comment:
should we add a info logging statement here to allert users that they are on the sandbox? would be helpful for me as i'm playing with ~20 configuration files ...